### ISSUE TITLE
Add a user option to block the boost button in case of missing ALT text

### ISSUE THREAD

**Issue by D4llo** [mastodon/mastodon#33817]
### Pitch

Add a user option to block the boost button in case of missing ALT text. I did not find a dupl

The idea emerge from the following thread (in french) https://pouet.chapril.org/@GythaOgg@pipou.academy/113939136471914729

The client Fedilab does that

### Motivation

Allow user to avoid a miss-boost of non alt content. Encourage inclusive content and a11y on the network.
